test/unit/BibPage.test.js

```diff
@@ -67,17 +68,12 @@ describe('BibPage', () => {
       expect(isAeonLink(resource.url)).to.be.true;
     });
 
-    it('should not include an Aeon link in top BibDetails', () => {
+    it('should not include an Aeon link in section of bib page', () => {
       const topBibComp = page.findWhere(
         (node) =>
-          node.type() === BibDetails && !node.prop('additionalData').length,
+          node.type() === ElectronicResources
       );
-      expect(topBibComp.type()).to.equal(BibDetails);
-      expect(
-        topBibComp.findWhere(
-          (el) => el.type() === 'dt' && el.text() === 'Electronic Resource',
-        ).length,
-      ).to.equal(1);
+      expect(topBibComp.type()).to.equal(ElectronicResources);
       expect(topBibComp.prop('electronicResources')).to.have.lengthOf(1);
```

Comment:
Shouldn't this be equal to 0 to match the description of the test?

I'd write a test to make sure that the "Available Online" text is displayed along with a link when there is an electronic resource.

Reply:
Not sure about line 77. That was there when i started working on this. Using the debugger, I see that the link that is rendered is a hathitrust link and not an aeon link. I assume the bib in question has aeon links and one other electronic resource url, and the length of 1 indicates that the aeon links are not being rendered.